addons/l10n_es/data/template/account.tax-es_common.csv

```diff
@@ -1,523 +1,524 @@
-"","","","","","","","","","","","tax","l10n_es.mod_303_31","invoice","account_common_472","100"
-"","","","","","","","","","","","tax","l10n_es.mod_303_13","invoice","account_common_477","-100"
-"","","","","","","","","","","","base","l10n_es.mod_303_40,l10n_es.mod_303_14_purchase","refund","","100"
-"","","","","","","","","","","","tax","l10n_es.mod_303_41","refund","account_common_472","100"
-"","","","","","","","","","","","tax","l10n_es.mod_303_15","refund","account_common_477","-100"
-"account_tax_template_p_iva10_isp_bi","","","","purchase","IVA 10% ISP (bienes de inversión)","10.0","percent","tax_group_iva_10","","","base","l10n_es.mod_303_30,l10n_es.mod_303_12","invoice","","100"
-"","","","","","","","","","","","tax","l10n_es.mod_303_31","invoice","account_common_472","100"
-"","","","","","","","","","","","tax","l10n_es.mod_303_13","invoice","account_common_477","-100"
-"","","","","","","","","","","","base","l10n_es.mod_303_40,l10n_es.mod_303_14_purchase","refund","","100"
-"","","","","","","","","","","","tax","l10n_es.mod_303_41","refund","account_common_472","100"
-"","","","","","","","","","","","tax","l10n_es.mod_303_15","refund","account_common_477","-100"
-"account_tax_template_p_iva21_isp_bi","","","","purchase","IVA 21% ISP (bienes de inversión)","21.0","percent","tax_group_iva_21","","","base","l10n_es.mod_303_30,l10n_es.mod_303_12","invoice","","100"
-"","","","","","","","","","","","tax","l10n_es.mod_303_31","invoice","account_common_472","100"
-"","","","","","","","","","","","tax","l10n_es.mod_303_13","invoice","account_common_477","-100"
-"","","","","","","","","","","","base","l10n_es.mod_303_40,l10n_es.mod_303_14_purchase","refund","","100"
-"","","","","","","","","","","","tax","l10n_es.mod_303_41","refund","account_common_472","100"
-"","","","","","","","","","","","tax","l10n_es.mod_303_15","refund","account_common_477","-100"
-"account_tax_template_p_rp19","","","","purchase","Retenciones 19% (préstamos)","-19.0","percent","tax_group_retenciones_19","","","base","","invoice","",""
-"","","","","","","","","","","","tax","","invoice","account_common_4751",""
-"","","","","","","","","","","","base","","refund","",""
-"","","","","","","","","","","","tax","","refund","account_common_4751",""
-"account_tax_template_p_rrD19","","","","purchase","Retenciones 19% (reparto de dividendos)","-19.0","percent","tax_group_retenciones_19","","","base","","invoice","",""
-"","","","","","","","","","","","tax","","invoice","account_common_4751",""
-"","","","","","","","","","","","base","","refund","",""
-"","","","","","","","","","","","tax","","refund","account_common_4751",""
+"id","sequence","description","invoice_label","type_tax_use","name","description@es","amount","amount_type","tax_group_id","analytic","include_base_amount","repartition_line_ids/repartition_type","repartition_line_ids/tag_ids","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/factor_percent"

```

Comment:
```suggestion
"id","sequence","description","description@es","invoice_label","type_tax_use","name","amount","amount_type","tax_group_id","analytic","include_base_amount","repartition_line_ids/repartition_type","repartition_line_ids/tag_ids","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/factor_percent"
```
NBD: it would be nicer to insert the `description@es` column right after the `description` column.

Reply:
It's unfortunately not that easy to order columns in IDEs and we haven't followed any rules in many PR we already made so I think if we want that it would be easier to run a script on all locas once malb and me are done with tax taxonomy rework. Works for you ? :)